src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -141,6 +142,10 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             _settingsKey = await GetSettingsKeyAsync(CancellationToken.None);
             UserSettings settings = LoadSettings();
             InitializeFilterList(settings);
+            var isReadmeTabEnabled = await _nugetFeatureFlagService.IsFeatureEnabledAsync(NuGetFeatureFlagConstants.RenderReadmeInPMUI);
+            _packageDetail._packageMetadataControl.InitializedReadmePreviewViewModel(Model.Context.ServiceBroker);
```

Comment:
Is this a precise name? I don't think it's initializing Readme tab, but if it is, it should only be if the feature flag is true?